lib/resources/content/webpack.config.template.js

```diff
@@ -43,6 +43,7 @@ module.exports = ({ production, server, extractCss, coverage, analyze, karma } =
     // out-of-date dependencies on 3rd party aurelia plugins
     alias: { 'aurelia-binding': path.resolve(__dirname, 'node_modules/aurelia-binding') }
   },
+  node: { fs: 'empty' }, // Fixes issue: "Error: Can't resolve 'fs'" when running karma tests
```

Comment:
Sorry I don't know much about the karma config, feel free to modify the PR for the best solution.

Reply:
I am no master of karma either. I just don’t understand the nature of this recent webpack karma issue.
Personally I try to avoid both webpack and karma as much as possible :)